includes/Classifai/Admin/Update.php

```diff
@@ -7,7 +7,7 @@
 
 namespace Classifai\Admin;
 
-use Puc_v4_Factory;
+use YahnisElsts\PluginUpdateChecker\v5\PucFactory;
```

Comment:
@dkotter This can cause package conflict between two plugins if both use different versions. I suggest adding a custom prefix to vendor packages which we require in the product environment.

Reply:
So we originally decided not to use a tool like PHP Scoper / Mozart / strauss (see https://github.com/10up/classifai/pull/365) and just ensure we were using the same version across our plugins. I'm not sure adding namespacing would solve the issue we've run into so far, where if someone is using composer to manage their plugins, it won't allow you to install multiple versions of the same dependency. That said, I'm not opposed to adding namespacing here though I won't have time to look into this until next week and ideally we were hoping to get this change out in the next release (tentatively scheduled for sometime next week). I know you've done work with `strauss` in the past, if you have interest and time to look at integrating that (or something similar here) feel free to takeover this PR

Reply:
@dkotter I will create an issue and resolve it separately pr.